ocs_ci/ocs/resources/pod.py

```diff
@@ -3855,7 +3855,7 @@ def get_pod_used_memory_in_mebibytes(podname):
         podname: (str)  name of the pod to get used memory of it
 
     Returns:
-        memory_value: (int) the used memory of the pod in Mebibytes (MiB)
+        int:  the used memory of the pod in Mebibytes (MiB)
```

Comment:
probably a leftover.
this line should stay the same (memory_value: (int))

Reply:
This is written as per google doc string format. Also I had referred previously written functions https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/ui/page_objects/block_and_file.py#L172